libvast/test/data.cpp

```diff
@@ -199,8 +199,10 @@ TEST(evaluation) {
 }
 
 TEST(evaluation - pattern matching) {
-  CHECK(evaluate(pattern{"f.*o"}, relational_operator::equal, "foo"));
-  CHECK(evaluate("foo", relational_operator::equal, pattern{"f.*o"}));
+  CHECK(
+    evaluate(unbox(to<pattern>("/f.*o/")), relational_operator::equal, "foo"));
+  CHECK(
+    evaluate("foo", relational_operator::equal, unbox(to<pattern>("/f.*o/"))));
```

Comment:
Can we also test case-insensitive evaluation?

Reply:
I added case-insensitive evaluation in https://github.com/tenzir/vast/pull/2964/commits/44b5a18cd6f4c783564dfffa4c636c5b3d9bf21c